torch/quantization/ns/utils.py

```diff
@@ -227,3 +227,41 @@ def get_target_type_str(node: Node, gm: GraphModule) -> str:
         target_mod = getattr_from_fqn(gm, node.target)
         target_type = str(type(target_mod))
     return target_type
+
+def rekey_logger_info_on_node_name_of_model(
+    results: NSResultsType,
+    model_name: str,
+) -> NSResultsType:
+    """
+    Rekeys the layer name of a results dictionary to use node names
+    from `model_name`.
+
+    For example, transforms
+
+        {'base_op_1_0': {'node_output': {'model_a':
+          [{'ref_node_name': 'linear1', ...}]}}}
+
+    into
+
+        {'linear1': {'node_output': {'model_a':
+          [{'ref_node_name': 'linear1', ...}]}}}
```

Comment:
Is it possible to keep the old layer name in the result? I can't think of specific use case now, but feel it might be useful during debugging.

Reply:
Since we already use ref_node_name as layer name, does it make sense to remove "ref_node_name" field from the result to reduce duplication?

Reply:
> Is it possible to keep the old layer name in the result? I can't think of specific use case now, but feel it might be useful during debugging.

Sorry seems we have "prev_node_name" in the result, please ignore this comment.

Reply:
> Is it possible to keep the old layer name in the result? I can't think of specific use case now, but feel it might be useful during debugging.

yes, the old layer name is unchanged in the result

> Since we already use ref_node_name as layer name, does it make sense to remove "ref_node_name" field from the result to reduce duplication?

hmm, so `ref_node_name` of one of the models is used as layer_name.  Might be good to keep it in the result as well, just to not change the keys of what is stored in the result.